run_dir/static/css/status.css

```diff
@@ -40,13 +40,13 @@ multiple places and typically control typography, or other common elements.
 	overflow: hidden;
 }
 
-#project_quotas td {
+.table-disk-quotas td {
```

Comment:
This selector is being overridden by the bootstrap default:
```css
.table>tbody>tr>td
```
(bootstrap being too specific is one of the annoyances being fixed in v4).
Changing this line to this should fix it:
```css
.table-disk-quotas > tbody > tr > td {
```